examples/04-admin/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "admin"
+version = "0.1.0"
+authors = ["Fluvio Contributors <team@fluvio.io>"]
+edition = "2018"
+publish = false
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+fluvio = { path = "../../src/client" }
+futures = "0.3.0"
+async-std = { version = "1.8.0", default-features = false,features = ["attributes"] }
+
+tracing = "0.1"
+tracing-subscriber = "0.2"
+tracing-opentelemetry = { version = "0.10" }
```

Comment:
What is your opinion on keeping or deleting this example? I mostly created it just to have an example program to run, not really intending to keep it. But it might be a good demonstration of some admin APIs. I would be just as happy to keep it or delete it, up to you.

Reply:
don't mind keep this as long

Reply:
I removed the tracing/telemetry stuff from the admin example and slimmed it down to focus on the watch streams. I think it's in a good shape to keep now.